reference-architecture/gcp/ansible/playbooks/openshift-post.yaml

```diff
@@ -11,7 +11,7 @@
       type: pd-ssd
   tasks:
   - name: set node selector for openshift-infra namespace
-    command: oc annotate --overwrite namespace openshift-infra openshift.io/node-selector='role=infra'
+    shell: PATH="${PATH}:/usr/local/bin:/usr/local/sbin"; oc annotate --overwrite namespace openshift-infra openshift.io/node-selector='role=infra'
```

Comment:
This is ok, but the "standard way" is to set individual components node selectors as:
* openshift_metrics_cassandra_nodeselector
* openshift_metrics_hawkular_nodeselector
* openshift_metrics_heapster_nodeselector

Reply:
Sure, I'm doing that here [1]. The namespace selector is useful when you need to do some manual tweaks later. See the discussion about this here [2]. Thanks for the review.

[1] https://github.com/openshift/openshift-ansible-contrib/blob/master/reference-architecture/gcp/ansible/playbooks/openshift-installer-common-vars.yaml#L26
[2] https://github.com/openshift/openshift-ansible-contrib/issues/453